assets/css/sensei-course-theme/base.scss

```diff
@@ -1,16 +1,29 @@
-:root {
-	--primary-color: var(--sensei-course-theme-primary-color, #30968B);
-	--primary-contrast-color: var(--sensei-course-theme-button-text-color, #FFFFFF);
-	--gray: #787C82;
+
+body {
+	--primary-color: var(--wp--preset--color--primary, #30968B);
```

Comment:
Oh looks like that customizer is actually broken and the color doesn't get saved properly. Will be looking at that in #4589

Reply:
I debugged the `get_theme_mod`, and it seems related to the filter
`add_filter( 'pre_option_stylesheet', [ $this, 'theme_stylesheet' ] );`

When it tries to get the theme_mod, it uses `get_option( 'stylesheet' )` to mount the option name. It means that the customizer is saving it for the selected theme, but when we're getting it, it's trying to get from the `sensei-course-theme`.

I think it would be nice to fix it before merging this, just in case we don't have enough time until the deadline to complete the https://github.com/Automattic/sensei/issues/4589, so we don't break it. Does it make sense?

Reply:
Found an easy way to use options instead of theme_mods for these variables, which doesn't depend on the theme. Just opening a PR for #4589 which will have the fix :)